Resolve table updating issues (locally, but possibly on production too?)

aileenzeng opened the pull request:
Could resolve #360 and possibly #366 if lucky?

I created a `dockerignore` file that excludes the `website/migrations` folder. If I remember correctly from my meeting with Jason a few weeks ago, the state of the `0001_initial.py` file seemed to be ahead of the actual database. I think this meant that Django thought it had already applied its migrations when it hadn't.

My hunch is that if we exclude the migrations folder, we'll be able to make sure that Django/the databases are in sync with each other. I tried it locally and it seemed to work fine (I was able to add/remove fields in `models.py` without any problems), and it didn't slow down any of the `docker build` or `docker run` steps.

jonfroehlich commented:
@aileenzeng, how do you suggest we test this? I'm inclined to just do the merge and try to fix any problems as they arise. Thoughts?

aileenzeng commented:
My general process for local testing was to get the database out of sync. I can't remember exactly how I did it, but I think I removed the `.dockerignore` file, made modifications to `models.py`, and checked out another branch. When I checked out the other branch, I rebuilt/reran the Docker container, and got a `column does not exist` error. After I added the `dockerignore` back, it seemed like the problem was fixed.

I think maybe it could use more local testing since I haven't had a chance to do it very thoroughly, since I sometimes have difficulty reproducing the error.

I can look into this more later and get a more detailed writeup (and maybe more specific guidelines for testing)?
